addons/website_sale/views/templates.xml

```diff
@@ -2797,18 +2812,19 @@
                     </div>
                 </t>
             </div>
-            <h3 class="mb-4">Confirm order</h3>
-            <div id="address_on_payment" class="mb-4">
-                <t t-call="website_sale.address_on_payment"/>
-            </div>
-            <div class="oe_structure clearfix mt-3" id="oe_structure_website_sale_payment_1"/>
```

Comment:
same comment here.
You should keep the div there, and add a new one if you need another one :eyes: